### DISCUSSION TITLE
Badge

### DISCUSSION THREAD

**Discussion by drknzz** [drknzz/GitHub-Achievements#14] [Q&A] [answered]
❓

**Comment by TiwariAbhishek23** [comment, marked as answer]
Hi, I want to earn my first badge, Galaxy brain. Help me, please.

**Reply by drknzz** [reply]
Hey, you need 2 answers to get the badge. I just marked your answer in this discussion, so if you answer any other discussion in this repository I will mark it as well do you can get the badge :)

**Reply by TiwariAbhishek23** [reply]
Yes, Thanks a lot, @drknzz.
This is my LinkedIn if you want, then we can connect - https://www.linkedin.com/in/abhishek-tiwari-5ba03422b/

**Comment by TiwariAbhishek23** [comment]
> ❓

Is it right, or am I doing something wrong?
